api/managementpb/dbaas/xtradb_cluster.proto

```diff
@@ -103,6 +103,8 @@ message ListXtraDBClustersResponse {
     RunningOperation operation = 3;
     // Cluster parameters.
     XtraDBClusterParams params = 4;
+    // Paused is true when cluster is suspended.
+    bool paused = 5;
```

Comment:
IMHO, it's better to handle this on backend side and just return new XtraDBClusterState, WDYT?

Reply:
:+1:

Reply:
Is it even a separate state on the operator level? Or that's just scaling to zero?

Reply:
Yes, we can do this to make UI more simple.
But we go away from what we really have in k8s and mix statuses with the other conception.

@AlekSi, no it is not separate state on operator level, this is scaling it to zero.
```
als@mbp percona-xtradb-cluster-operator % kubectl get pxc/my-test-pxc -o json | jq .status.state
"ready"
als@mbp percona-xtradb-cluster-operator % kubectl get pod                                       
NAME                                               READY   STATUS    RESTARTS   AGE
percona-xtradb-cluster-operator-5c99579f95-k82p4   1/1     Running   0          16h
als@mbp percona-xtradb-cluster-operator % kubectl get pxc/my-test-pxc -o json | jq .spec.pause  
true
als@mbp percona-xtradb-cluster-operator % kubectl get pvc | grep my-test-pxc                     
datadir-my-test-pxc-pxc-0   Bound    pvc-11a1e4be-4f71-458a-914d-32f5da1e9431   6Gi        RWO            gp2            29m
datadir-my-test-pxc-pxc-1   Bound    pvc-40efe271-814d-4edc-bd03-74c38dfbb0a0   6Gi        RWO            gp2            28m
datadir-my-test-pxc-pxc-2   Bound    pvc-345fd6d1-0f39-44b4-8f22-01c06934bfe3   6Gi        RWO            gp2            27m
```

Reply:
We use separate cluster states in our API, so I think we can add one more state to our states list.